app/Controllers/apiController.php

```diff
@@ -10,7 +10,7 @@ class FreshRSS_api_Controller extends FreshRSS_ActionController {
 	 * Return an error message, or `false` if no error.
 	 */
 	public static function updatePassword($apiPasswordPlain) {
-		$username = Minz_Session::param('currentUser');
+		$username = Minz_Session::param(CURRENT_USER);
```

Comment:
We use it so much that I believe a new function `currentUser`, e.g. in `FreshRSS_Context` would be worth it.

Reply:
Done